### PR TITLE
Add project description for llm benchmark

### PR DESCRIPTION
**Pull request by ChristopherSpelt**
# First version of LLM-Benchmark project description

## Checklist

Please check all the boxes that apply to this pull request using "x":

- [x]  I have tested the changes locally and verified that they work as expected.
- [x]  I have followed the project's coding conventions and style guidelines.
- [x]  I have rebased my branch onto the latest commit of the main branch.
- [x]  I have squashed or reorganized my commits into logical units.
- [x]  I have read, understood and agree to the Developer Certificate of Origin below, which this project utilizes.

## Developer Certificate of Origin

Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

File: docs/Projects/LLM-benchmarks/project_description.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# LLL-Benchmark project
+
+Large Languages Models (LLM's) are becoming increasingly popular in assisting people in
+a variety of tasks. These tasks include, but are not limited to, information retrieval, assisting with
+coding and essay writing. In the context of the government, tasks can include for example supporting FOIA
```
Comment (laurensWe) [inline docs/Projects/LLM-benchmarks/project_description.md:5]: I think the abbreviation FOIA is maybe not so well known and therefore weird to introduce it here in one go maybe

File: docs/Projects/LLM-benchmarks/project_description.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# LLL-Benchmark project
+
+Large Languages Models (LLM's) are becoming increasingly popular in assisting people in
```
Comment (laurensWe) [inline docs/Projects/LLM-benchmarks/project_description.md:3]:
```suggestion
Large Languages Models (LLMs) are becoming increasingly popular in assisting people in
```

Comment (laurensWe) [reply docs/Projects/LLM-benchmarks/project_description.md:3]: I think this is the more used plural form of LLM

File: docs/Projects/LLM-benchmarks/project_description.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# LLL-Benchmark project
```
Comment (laurensWe) [inline docs/Projects/LLM-benchmarks/project_description.md:1]:
```suggestion
# LLM-Benchmark project
```

File: docs/Projects/LLM-benchmarks/project_description.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
```
Comment (laurensWe) [inline docs/Projects/LLM-benchmarks/project_description.md:2]:
```suggestion
## Context
```

File: docs/Projects/LLM-benchmarks/project_description.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+Large Languages Models (LLM's) are becoming increasingly popular in assisting people in
+a variety of tasks. These tasks include, but are not limited to, information retrieval, assisting with
+coding and essay writing. In the context of the government, tasks can include for example supporting FOIA
+requests and aiding in answering questions of citizens.
+
+While the potential benefit of using LLM's is large, there are also significant risks.
+Basically an LLM is just a next token predictor, which bases its predictions on the user input (context) and
```
Comment (laurensWe) [inline docs/Projects/LLM-benchmarks/project_description.md:9]: I like it, but at the same time I am doubting why specific technologies should be introduced for the project itself, just putting my thought out there

File: docs/Projects/LLM-benchmarks/project_description.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+is no guarantee on the quality and correctness of the output. Moreover, due to
+bias in the training data, LLM's can have bias in their output, despite best efforts
+to mitigate this. Additionally, we have human values that we expect LLM's to be aligned with.
+Certainly, within the context of a government, we should take utmost care not to discriminate.
+To assess the quality, correctness, bias and alignment with human values of a LLM one can perform
+benchmarks.
+
+## The project
+
+The LLM-benchmark project of the AI-Validation Team of the Ministry of the Interior and Kingdom
```
Comment (laurensWe) [inline docs/Projects/LLM-benchmarks/project_description.md:20]:
```suggestion
The LLM-benchmark project of the AI-Validation Team 
```

Comment (laurensWe) [reply docs/Projects/LLM-benchmarks/project_description.md:20]: This is implied by this being in this repository and also is just terribly long

File: docs/Projects/LLM-benchmarks/project_description.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+Relations of the Government of the Netherlands aims to create a platform where LLM's can be measured
```
Comment (laurensWe) [inline docs/Projects/LLM-benchmarks/project_description.md:21]:
```suggestion
 aims to create a platform where LLMs can be measured
```

File: docs/Projects/LLM-benchmarks/project_description.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+across a wide range of benchmarks. Both LLM's and the benchmarks can be configured by users of the platform.
```
Comment (laurensWe) [inline docs/Projects/LLM-benchmarks/project_description.md:22]: I miss the piece here still what the project is exactly, you state here users can configure them, this does imply that it is some kind of platform where they can be ran.

File: aivalidation_dictionary.txt [hunk 9]
```diff
@@ -32,6 +33,8 @@ interpretability
 ITerm
 Kubeflow
 Laurens
+LLM
+LLM's
```
Comment (uittenbroekrobbert) [inline aivalidation_dictionary.txt:37]: The plural of LLMs seems to be LLMs: https://en.wiktionary.org/wiki/LLMs#English

File: docs/Projects/LLM-benchmarks/project_description.md (hunk 1 above)
Comment (uittenbroekrobbert) [reply docs/Projects/LLM-benchmarks/project_description.md:5]: Agreed, I would have to Google it

File: docs/Projects/LLM-benchmarks/project_description.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
+# LLL-Benchmark project
+
+Large Languages Models (LLM's) are becoming increasingly popular in assisting people in
+a variety of tasks. These tasks include, but are not limited to, information retrieval, assisting with
+coding and essay writing. In the context of the government, tasks can include for example supporting FOIA
+requests and aiding in answering questions of citizens.
+
+While the potential benefit of using LLM's is large, there are also significant risks.
```
Comment (uittenbroekrobbert) [inline docs/Projects/LLM-benchmarks/project_description.md:8]:
```suggestion
While the potential benefit of using LLMs is large, there are also significant risks.
```

File: aivalidation_dictionary.txt (hunk 9 above)
Comment (uittenbroekrobbert) [inline aivalidation_dictionary.txt:37]:
```suggestion
LLMs
```

File: docs/Projects/LLM-benchmarks/project_description.md (hunk 5 above)
Comment (uittenbroekrobbert) [inline docs/Projects/LLM-benchmarks/project_description.md:9]:
```suggestion
Basically a LLM is just a next token predictor, which bases its predictions on the user input (context) and
```

File: docs/Projects/LLM-benchmarks/project_description.md [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+Basically an LLM is just a next token predictor, which bases its predictions on the user input (context) and
+on compressed information seen during training (LLM parameters); hence there
+is no guarantee on the quality and correctness of the output. Moreover, due to
+bias in the training data, LLM's can have bias in their output, despite best efforts
```
Comment (uittenbroekrobbert) [inline docs/Projects/LLM-benchmarks/project_description.md:12]:
```suggestion
bias in the training data, LLMs can have bias in their output, despite best efforts
```

File: docs/Projects/LLM-benchmarks/project_description.md [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+to mitigate this. Additionally, we have human values that we expect LLM's to be aligned with.
```
Comment (uittenbroekrobbert) [inline docs/Projects/LLM-benchmarks/project_description.md:13]:
```suggestion
to mitigate this. Additionally, we have human values that we expect LLMs to be aligned with.
```

File: docs/Projects/LLM-benchmarks/project_description.md (hunk 8 above)
Comment (uittenbroekrobbert) [inline docs/Projects/LLM-benchmarks/project_description.md:22]:
```suggestion
across a wide range of benchmarks. Both LLMs and the benchmarks can be configured.
```

File: docs/Projects/LLM-benchmarks/project_description.md [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+To assess the quality, correctness, bias and alignment with human values of a LLM one can perform
+benchmarks.
+
+## The project
+
+The LLM-benchmark project of the AI-Validation Team of the Ministry of the Interior and Kingdom
+Relations of the Government of the Netherlands aims to create a platform where LLM's can be measured
+across a wide range of benchmarks. Both LLM's and the benchmarks can be configured by users of the platform.
+The intended goal of this project is to give government organizations, citizens and companies insight in the various
+LLM's and their quality, correctness, bias and alignment with human values. The project also encompasses a dashboard with
```
Comment (uittenbroekrobbert) [inline docs/Projects/LLM-benchmarks/project_description.md:24]:
```suggestion
LLMs and their quality, correctness, bias and alignment with human values. The project also encompasses a dashboard with
```

File: docs/Projects/LLM-benchmarks/project_description.md [hunk 14]
```diff
@@ -0,0 +1,26 @@
...
+uploaded LLM's and their performance on uploaded benchmarks. With this platform we aim to enhance
```
Comment (uittenbroekrobbert) [inline docs/Projects/LLM-benchmarks/project_description.md:25]:
```suggestion
uploaded LLMs and their performance on uploaded benchmarks. With this platform we aim to enhance
```

File: docs/Projects/LLM-benchmarks/project_description.md [hunk 15]
```diff
@@ -0,0 +1,26 @@
...
+public trust in the usage of LLM's and expose potential bias that exists within LLM's.
```
Comment (uittenbroekrobbert) [inline docs/Projects/LLM-benchmarks/project_description.md:26]:
```suggestion
public trust in the usage of LLMs and expose potential bias that exists within LLMs.
```

File: docs/Projects/LLM-benchmarks/project_description.md (hunk 5 above)
Comment (anneschuth) [reply docs/Projects/LLM-benchmarks/project_description.md]: What do you mean with specific technologies?

Comment (anneschuth) [reply docs/Projects/LLM-benchmarks/project_description.md]: Wait, I think it should be "an LLM" since LLM is pronounced as "el-el-em".

**Review by anneschuth — DISMISSED** [review]
Nice!

**Review by uittenbroekrobbert — APPROVED** [review]

**Comment by robbertbos** [comment]
@ChristopherSpelt nice! I would propose to scope the LLM Benchmarks to languages related to Dutch society. What do you think?

**Pull request merged**